docs/resource-sharing/os-backfill-containers.md

```diff
@@ -201,29 +183,40 @@ add the following to your `docker run` command:
 
 where `<X>` is the number of CPUs you want to allow jobs to use.
 
-The `NUM_CPUS` environment variable will tell the pilot not to offer more than the given number of CPUs to jobs;
-the `--cpus` argument will tell Docker not to allocate more than the given number of CPUs to the container.
-
-Both options are necessary for optimal behavior.
+The `NUM_CPUS` environment variable tells the pilot not to offer more than the given number of CPUs to jobs;
+the `--cpus` argument tells Docker not to allocate more than the given number of CPUs to the container.
 
 
-### Limiting memory
+#### Memory
 
 To limit the total amount of memory available to jobs, add the following to your `docker run` command:
+
 ```
     -e MEMORY=<X> --memory=$(( (<X> + 100) * 1024 * 1024 )) \
 ```
 
 where `<X>` is the total amount of memory (in MB) you want to allow jobs to use.
 
-The `MEMORY` environment variable will tell the pilot not to offer more than the given amount of memory to jobs;
-the `--memory` argument will tell Docker to kill the container if its total memory usage exceeds the given number.
-
-Both options are necessary for optimal behavior.
+The `MEMORY` environment variable tells the pilot not to offer more than the given amount of memory to jobs;
+the `--memory` argument tells Docker to kill the container if its total memory usage exceeds the given number.
 
 !!! tip "Allocating additional memory"
     Note that the above command will allocate 100 MB more memory to the container.
     The pilot will place jobs on hold if they exceed their requested memory,
     but it may not notice high memory usage immediately.
-    In addition, the processes that manage jobs also use some amount of memory.
-    Therefore it is important to give the container some extra room.
+    Additionally, the processes that manage jobs also use some amount of memory.
+    Therefore, it is important to give the container some extra room.
+
+Best Practices
+--------------
+
+We recommend redeploying backfill containers at least every 72 hours.
```

Comment:
```suggestion
We recommend pulling new versions of backfill containers at least every 72 hours.
```
or something -- remind people that an explicit pull is necessary and docker run won't just get the latest version each time.